content/src/ports/activeEntities.ts

```diff
@@ -0,0 +1,241 @@
+import { Entity, EntityId, EntityType, Pointer } from 'dcl-catalyst-commons'
+import LRU from 'lru-cache'
+import { EnvironmentConfig } from '../Environment'
+import { mapDeploymentsToEntities } from '../logic/deployments'
+import { getDeployments } from '../service/deployments/deployments'
+import { AppComponents } from '../types'
+
+export const NOT_ACTIVE = 'NONE'
+export type EntityCacheResult = Entity | typeof NOT_ACTIVE
+export type PointerToEntityId = EntityId | typeof NOT_ACTIVE
+
+const isEntityPresent = (result: EntityCacheResult | undefined): result is Entity => result !== NOT_ACTIVE
+const isPointingToEntity = (result: PointerToEntityId | undefined): result is EntityId => result !== NOT_ACTIVE
+
+export type ActiveEntities = {
+  /**
+   * Retrieve active entities that are pointed by the given pointers
+   * Note: result is cached, even if the pointer has no active entity
+   */
+  withPointers(pointers: Pointer[]): Promise<Entity[]>
+  /**
+   * Retrieve active entities by their ids
+   * Note: result is cached, even if the id has no active entity
+   */
+  withIds(entityIds: EntityId[]): Promise<Entity[]>
+  /**
+   * Save entityId for given pointer and store the entity in the cache,
+   * useful to retrieve entities by pointers
+   */
+  update(pointers: Pointer[], entity: EntityCacheResult): void
```

Comment:
"Clearing" the pointers means importing the `NOT_ACTIVE` type from this module and passing that as a parameter to the update function. `NOT_ACTIVE` feels like an internal object to me (as we're not returning it anywhere, just omitting the pointers that does not have an active entity)

What do you think about having two methods to interact with the `ActiveEntities`? 

I was thinking in something sooomehow like:
`update(pointers: Pointer[], entity: Entity[])`
`clear(pointers)`